### ISSUE TITLE
The Shore VR (1834620)

### ISSUE THREAD

**Issue by Patola** [ValveSoftware/Proton#8338]
# Compatibility Report
- Name of the game with compatibility issues: The Shore VR
- Steam AppID of the game: 1834620

## System Information
- GPU: RX 7900 XTX
- Video driver version: mesa 24.3.2
- Kernel version: 6.12.6-273-tkg-eevdf
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Patola/d35f7df845a09b7409d6505c3145a2be
- Proton version: 9.0-4

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 

Game doesn't start. Instead it crashes to the desktop with an unreal engine 4 crash reporter:
![image](https://github.com/user-attachments/assets/191734c2-5b55-40b9-93b7-cb13ea5e2d6c)

## Reproduction
Just run the game.

Proton log is attached.
[steam-1834620.log](https://github.com/user-attachments/files/18236729/steam-1834620.log)

**Comment by Patola** [comment]
Notes:
* The game has a pancake version that works on Linux
* With VR off, if I run the game by pressing play, the graphics appear on the screen without a crash. But of course it's unplayable.

**Comment by kisak-valve** [comment]
Hello @Patola, these look like some lines of interest from your Proton log:
```
err:vkd3d-proton:vkd3d_check_extensions: Required user device extension "VK_KHR_external_memory_fd" is not supported.
err:vkd3d-proton:vkd3d_check_extensions: Required user device extension "VK_KHR_external_semaphore_fd" is not supported.
err:vkd3d-proton:vkd3d_check_extensions: Required user device extension "VK_WINE_openxr_device_extensions" is not supported.
[...]
err:vrclient:vrclient_translate_texture_d3d12 Unsupported DXGI format 0x5a.
```

**Comment by kisak-valve** [comment]
https://github.com/ValveSoftware/Proton/commit/8c3e86c52c2baf75e04ea5d9cdd9b1c6c83161aa might help a bit with the first couple lines, but I expect that the last line is where the game is falling over.

**Comment by Patola** [comment]
Indeed. It's off that the game requires these extensions. The first two are available on the driver, only the last one isn't. And there is a number of UE4 VR games that also crash like this one. In this case should I create a broader issue to account for every game that crashes like this (if I do that, not sure I will find them), or just cross-reference in a new compatibility issue?

```
[16:41] [5051] [patola@risadinha /tmp]% vulkaninfo 2>/dev/null | grep -E 'VK_KHR_external_memory_fd|VK_KHR_external_semaphore_fd|VK_WINE_openxr'
        VK_KHR_external_memory_fd                     : extension revision 1
        VK_KHR_external_semaphore_fd                  : extension revision 1
[16:41] [5052] [patola@risadinha /tmp]% 
```
